src/generated/models/AppInfo.ts

```diff
@@ -294,7 +294,8 @@ export interface AppInfo {
  */
 export const AppInfoTypeEnum = {
     Complete: 'complete',
-    Flex: 'flex'
+    Flex: 'flex',
+    FlexV2: 'flex_v2'
```

Comment:
Is that support live on prod? Also how would I go about regenerating this?

Reply:
You can download the most recent SDK spec from the Github action and run it with the `generate.sh` file. 

It would be a good idea for us to create a Github action just to regenerate SDKs.